hazelcast/src/main/java/com/hazelcast/replicatedmap/impl/operation/PutAllOperation.java

```diff
@@ -36,7 +34,7 @@
 /**
  * Puts a set of records to the replicated map.
  */
-public class PutAllOperation extends Operation implements IdentifiedDataSerializable {
+public class PutAllOperation extends AbstractSerializableOperation {
```

Comment:
I wanted to have it like this, but the AbstractReplicatedMapOperation has some fields that are not used everywhere, so actually AbstractReplicatedMapOperation should extend AbstractSerializableOperation.